AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -68,6 +68,10 @@ extension ApplePayComponent {
 
         /// A funding source supported by the merchant. If `nil`, the transaction allows both credit and debit cards.
         public var merchantCapability: CardFundingSource?
+        
+        /// The payment request object needed for Apple Pay. Must contain all the required fileds
+        /// such as `merchantIdentifier`, `summaryItems`, `currencyCode`, and `countryCode`.
+        public private(set) var paymentRequest: PKPaymentRequest?
```

Comment:
Does this only have to be public because of checking in tests?
```suggestion
        internal private(set) var paymentRequest: PKPaymentRequest?
```

Reply:
Fixed in cc142c422894338fb2a6e72c2cce560a45da754f